integration-tests/gradle/src/test/resources/conditional-dependencies/ext-quarkus-a/deployment/build.gradle

```
@@ -0,0 +1,22 @@
+plugins {
+    id 'java-library'
+    id 'maven-publish'
+}
+
+dependencies {
+    implementation enforcedPlatform("${quarkusPlatformGroupId}:${quarkusPlatformArtifactId}:${quarkusPlatformVersion}")
+    implementation 'io.quarkus:quarkus-core-deployment'
+    implementation project(':ext-quarkus-a:runtime')
+}
+
+publishing {
+    publications {
+        mavenJave(MavenPublication) {
```

Just saw this by chance:
```suggestion
        mavenJava(MavenPublication) {
```
But then again I have no clue what I'm talking about. 😄

---

nice catch!